### PR TITLE
Update the SapMachine 11.0.9 to 11.0.9.1

### PR DESCRIPTION
**Pull request by poad**
## Removes
- 11.0.9

## Added
 - 11.0.9.1

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/java/SapMachineMigrations.scala [hunk 1]
```diff
@@ -65,57 +65,53 @@ class SapMachineMigrations {
...
-      MacOSX  -> "osx-x64_bin.tar.gz",
-      Windows -> "windows-x64_bin.zip"
+      Windows -> "windows-x64_bin.zip",
+      MacOSX  -> "osx-x64_bin.tar.gz"
     ).map {
         case (platform, binary) =>
-          toVersion(platform, binary, "15.0.1")
+          Version(
+            "java",
+            "11.0.9.1-sapmchn",
```
Comment (eddumelendez) [inline src/main/scala/io/sdkman/changelogs/java/SapMachineMigrations.scala:110]: I think we should keep

```suggestion
            "11.0.9-sapmchn",
```

File: src/main/scala/io/sdkman/changelogs/java/SapMachineMigrations.scala [hunk 2]
```diff
@@ -65,57 +65,53 @@ class SapMachineMigrations {
...
+            s"https://github.com/SAP/SapMachine/releases/download/sapmachine-11.0.9/sapmachine-jdk-11.0.9_$binary",
+            platform,
+            Some(AdoptOpenJDK)
```
Comment (eddumelendez) [inline src/main/scala/io/sdkman/changelogs/java/SapMachineMigrations.scala:113]:
```suggestion
            Some(SAP)
```

File: src/main/scala/io/sdkman/changelogs/java/SapMachineMigrations.scala [hunk 3]
```diff
@@ -65,57 +65,53 @@ class SapMachineMigrations {
...
+    id = "0023-add_sapmchn_jdk_11.0.9_1",
+    author = "poad"
   )
-  def migrate0021(implicit db: MongoDatabase) =
+  def migrate0023(implicit db: MongoDatabase) =
     Map(
       Linux64 -> "linux-x64_bin.tar.gz",
-      MacOSX  -> "osx-x64_bin.tar.gz",
-      Windows -> "windows-x64_bin.zip"
+      Windows -> "windows-x64_bin.zip",
```
Comment (eddumelendez) [inline src/main/scala/io/sdkman/changelogs/java/SapMachineMigrations.scala:104]: order should be alphabetically

**Review by eddumelendez — CHANGES_REQUESTED** [review]

File: src/main/scala/io/sdkman/changelogs/java/SapMachineMigrations.scala [hunk 4]
```diff
@@ -65,57 +65,53 @@ class SapMachineMigrations {
...
       Linux64 -> "linux-x64_bin.tar.gz",
       MacOSX  -> "osx-x64_bin.tar.gz",
       Windows -> "windows-x64_bin.zip"
     ).map {
         case (platform, binary) =>
-          toVersion(platform, binary, "15.0.1")
+          Version(
+            "java",
+            "11.0.9-sapmchn",
+            s"https://github.com/SAP/SapMachine/releases/download/sapmachine-11.0.9/sapmachine-jdk-11.0.9_$binary",
```
Comment (eddumelendez) [inline src/main/scala/io/sdkman/changelogs/java/SapMachineMigrations.scala:111]: The right url is

```suggestion
            s"https://github.com/SAP/SapMachine/releases/download/sapmachine-11.0.9.1/sapmachine-jdk-11.0.9.1-1_$binary",
```

**Review by eddumelendez — APPROVED** [review]

**Comment by poad** [comment]
Thank you for reviewing this.
I'm sorry for the many errors .

**Comment by eddumelendez** [comment]
@poad no need to say sorry. thank you so much for your contributions!

**Pull request merged**